### DISCUSSION TITLE
What is the python function of process_tasks?

### DISCUSSION THREAD

**Discussion by kingstar1010** [community/community#37881] [Other Feature Feedback, Questions, & Ideas] [answered]
Who can help me?

**Comment by ghost** [comment, marked as answer]
Django Background Task is a databased-backed work queue for Django, loosely based around Ruby’s DelayedJob library. This project was adopted and adapted from this repo.

To avoid conflicts on PyPI we renamed it to django-background-tasks (plural). For an easy upgrade from django-background-task to django-background-tasks, the internal module structure were left untouched.

-Command
python manage.py process_tasks
python manage.py process_tasks&

**Reply by kingstar1010** [reply]
Thank you!
